nifi-extension-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/pom.xml

```diff
@@ -51,5 +51,11 @@
             <artifactId>lucene-backward-codecs</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-framework-core-api</artifactId>
```

Comment:
What is happening that is requiring a new dependency on the nifi-framework-core-api to be introduced here?  And the reference to compile - is it intended this API is copied into the NAR?